src/extension/extensionServer.ts

```diff
@@ -299,6 +309,8 @@ function requestSetup(args: any): any {
         target: args.target || "simulator",
     };
 
+    CommandExecutor.ReactNativeCommand = reactNativeGlobalCommandName;
```

Comment:
Duplicate of https://github.com/microsoft/vscode-react-native/pull/1093/files#diff-a0bb3c3c1809e7b5cf209c8263767d8fR302